integration/heapster_api_test.go

```diff
@@ -190,6 +207,31 @@ func getSchema(fm kubeFramework, svc *kube_api.Service) (*api_v1.TimeseriesSchem
 	return &timeseriesSchema, nil
 }
 
+func getModelMetrics(fm kubeFramework, svc *kube_api.Service, pod *kube_api.Pod) (*api_v1.MetricResultList, error) {
+	url := fmt.Sprintf("/api/v1/model/namespaces/%s/pod-list/%s/metrics/%s",
```

Comment:
I am perfectly fine with adding checks for the other endpoints in a future
PR. I just wanted to bring that up.

On Tue, Sep 15, 2015 at 3:19 PM, Vishnu Kannan <vishnuk@google.com> wrote:

> Fair point! I see value for all of them and in terms of priority, the
> bounding of methods might be the final thing we want to test, with the
> assumption that code reviews would have caught that.
>
> On Tue, Sep 15, 2015 at 3:11 PM, Marcin Wielgus <notifications@github.com>
> wrote:
>
>> In integration/heapster_api_test.go
>> <https://github.com/kubernetes/heapster/pull/558#discussion_r39572424>:
>>
>> > @@ -190,6 +207,31 @@ func getSchema(fm kubeFramework, svc *kube_api.Service) (*api_v1.TimeseriesSchem
>> >  	return &timeseriesSchema, nil
>> >  }
>> >
>> > +func getModelMetrics(fm kubeFramework, svc *kube_api.Service, pod *kube_api.Pod) (*api_v1.MetricResultList, error) {
>> > +	url := fmt.Sprintf("/api/v1/model/namespaces/%s/pod-list/%s/metrics/%s",
>>
>> With the E2E approach as here i can only check if the output is returned
>> at all and if it is sane, not if the endpoints are correctly bound to model
>> funcs.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kubernetes/heapster/pull/558/files#r39572424>.
>>
>
>